book/src/developers/contributing/releases/release_checklist.md

```diff
@@ -17,31 +17,29 @@ Make sure that version follows [semver](https://semver.org/) rules e.g (`0.2.0-a
 
 **For the time being, ALWAYS specify the `-alpha` suffix.**
 
-## Bump the version
+## Release the version
 
-- In github, open the page to [create the new release](https://github.com/ethereum/trin/releases/new).
-- In the tag, type out the version number that the new release bumps to and select "Create new tag".
-- Github should say "Excellent! This tag will be created from the target when you publish this release."
-- Click "Generate release notes"
-- Select "Set as a pre-release"
-- Add "Trin " to the beginning of the release title
-- Add any clarifying information that's helpful about the release
+We use automated github release workflow to create a new release.
+This will create a new release draft with the new tag and will build all the binaries and attach them to the release. 
 
-## Build the binary for release
-
-> ⚠️  **Under development**: This is an untested rough draft. Pair up when
-> releasing, to verify.
-
-Build the binary with:
-
-```sh
-cargo build --release
+1. Checkout and rebase local master to upstream
+```bash
+git checkout master
+git pull --rebase upstream master
```

Comment:
Just want to make sure that we're not missing any submodule commands here? eg. updating the submodule?

Reply:
We don't need to update the submodule when creating a tag.